### DISCUSSION TITLE
No transfer of IP

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#21] [Q&A]
says

https://github.com/commonhaus/foundation-draft/blob/main/governance/ip-policy.md
```
Intellectual Property Rights: CF membership does not grant or transfer intellectual property rights unless explicitly stated by this policy or a Project's license.
```

What does that mean concretely? What is the intent of that sentence

**Comment by ebullient** [comment]
This section is designed to protect the IP rights of individual contributors.

Unless there's a specific provision within a project's license or the CF's policy, _contributors_ maintain control and ownership of their IP even after contributing to a project under the CF. This is related to copyright statements, where contributors retain their copyright.

So, if my project says "Copyright 2023 Contributors to the XYZ project", that copyright is not transferred to the Commonhaus Foundation if/when project XYZ joins the foundation UNLESS that project says so.

**Comment by kenfinnigan** [comment]
This question is now resolved in the by-laws. If anything above is not resolved, please open a PR with suggested changes to the by-laws/docs.
